helloworld/lib/greet.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = function greet(name){
+  if (typeof name !== 'string') throw new Error('expected a string');
```

Comment:
Remember, the instructions on Lab 1 specifically also said that this function should ```return null``` if a string is not entered. I won't dock you points for this, but just a friendly reminder to read the specifics. :)